package.json

```diff
@@ -29,7 +29,7 @@
     "url":
       "https://github.com/relay-tools/relay-compiler-language-typescript/issues"
   },
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
```

Comment:
Was this intentional?

Reply:
that's a great question! looks like an artifact from something i tried & forgot to revert.
sorry for letting this go stale, there's no errors with the latest relay-runtime types, so i'm happy to close

Reply:
Let’s do that then 👍